docker/docker.go

```diff
@@ -28,8 +28,44 @@ import (
 
 type Port = types.Port
 
+// GetClient returns a docker client if all checks pass.
+// This function performs three checks:
+//   1. checks that docker is installed and running properly,
+//   2. checks that the user is not running docker toolbox.
+//   3. checks that the client api version is supported by the docker engine,
+func GetClient() (*client.Client, error) {
```

Comment:
I'm not sure if we should we export this.
The rest of the code is calling this package convenience wrappers, not getting a client and use it directly.

For `prune_test.go` I think it makes sense to use `docker.GetClient()` directly. Or move `ListNetworks` to this package if needed.

What are others' thoughts?

Reply:
it makes sense to move `ListNetworks` into docker package.
but I don't see a big problem with exporting `GetClient` also.

Reply:
Agree for moving `ListNetworks`, but I'd keep `GetClient` exposed even if so far there's no need to use it from other packages.

Reply:
Moved `ListNetworks` [here](https://github.com/src-d/engine/pull/428/commits/35b17e92040a95d26498a37e31d2b2d449965624).